src/ScottPlot5/ScottPlot5/Primitives/Label.cs

```diff
@@ -30,57 +30,11 @@ public class Label
     public PixelOffset ShadowOffset { get; set; } = new(3, 3);
 
     public bool AntiAliasBackground { get; set; } = true;
-    public bool AntiAliasText { get; set; } = true;
-
-    // TODO: use a class for cached typeface management
-
-    private SKTypeface? CachedTypeface = null;
-    private SKTypeface Typeface
-    {
-        get
-        {
-            if (CachedTypeface is not null)
-                return CachedTypeface;
-
-            if (FontFile is not null)
-                return FontStyle.CreateTypefaceFromFile(FontFile);
-
-            return FontStyle.CreateTypefaceFromName(FontName, Bold, Italic);
-        }
-    }
-
-    public void SetTypeface(SKTypeface typeface)
-    {
-        CachedTypeface = typeface;
-    }
-
-    private string _FontName = Fonts.Default;
-    public string FontName
-    {
-        get => _FontName;
-        set { _FontName = value; ClearCachedTypeface(); }
-    }
-
-    private string? _FontFile = null;
-    public string? FontFile
-    {
-        get => _FontFile;
-        set { _FontFile = value; ClearCachedTypeface(); }
-    }
-
-    private float _FontSize = 12;
-    public float FontSize
-    {
-        get => _FontSize;
-        set { _FontSize = value; ClearCachedTypeface(); }
-    }
-
-    private bool _Bold = false;
-    public bool Bold
-    {
-        get => _Bold;
-        set { _Bold = value; ClearCachedTypeface(); }
-    }
+    public bool AntiAliasText { get; set; } = true;

```

Comment:
Very nice, this cleans it up well

Reply:
Please explain your comment, I don’t quite understand what it means.